src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -550,9 +550,20 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
       ];
       if (item.reason) {
         this.hass.loadBackendTranslation("config", item.domain);
-        stateTextExtra = html`${this.hass.localize(
+        const stateTextError = this.hass.localize(
           `component.${item.domain}.config.error.${item.reason}`
-        ) || item.reason}`;
+        );
+        let stateTextReason: string | null = null;
+        if (item.reason_translation_key) {
+          this.hass.loadBackendTranslation("exceptions", item.domain);
```

Comment:
you will need to await this

Reply:
```
'await' expressions are only allowed within async functions and at the top levels of modules.ts(1308)
ha-config-integration-page.ts(532, 11): Did you mean to mark this function as 'async'?
```
Not sure what that means and also we have loading translations on line 552 which is not awaited so not sure I would understand the difference

Reply:
Yeah that seems wrong too, one way to fix it is:

```diff
@@ -549,10 +550,15 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
         `ui.panel.config.integrations.config_entry.state.${item.state}`,
       ];
       if (item.reason) {
-        this.hass.loadBackendTranslation("config", item.domain);
-        stateTextExtra = html`${this.hass.localize(
-          `component.${item.domain}.config.error.${item.reason}`
-        ) || item.reason}`;
+        const lokaliseProm = this.hass
+          .loadBackendTranslation("config", item.domain)
+          .then(
+            (localize) =>
+              localize(
+                `component.${item.domain}.config.error.${item.reason}`
+              ) || item.reason
+          );
+        stateTextExtra = html`${until(lokaliseProm, item.reason)}`;
       } else {
         stateTextExtra = html`
           <br />
```

Reply:
I changed somehow to your suggestion but I haven't tested if it works yet